system_tasks.yml

```diff
@@ -15,6 +15,26 @@
     - python-dev
     - gdal-bin
     - unzip
+    - python-apt
+
+# By default, Piecewise uses EFF's LetsEncrypt CA for obtaining SSL certificates.
```

Comment:
Yes. EFF's certbot tool, which automatically requests and obtains SSL certificates from the LetsEncrypt CA.